platform_main_feature.strongarm.inc

```diff
@@ -267,6 +274,13 @@ function platform_main_feature_strongarm() {
   $strongarm = new stdClass();
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'preprocess_css';
+  $strongarm->value = 0;
```

Comment:
Is this supposed to be on or off? [Looks like it was turned off](https://github.com/MorganDawe/platform_main_feature/commit/bd509d0eaf6238871d866d3ebc1326ea3936a490#diff-5a08a43b2c687be219ac25ecea6605d6R271) (in the second commit in this PR)...